packages/oas-to-har/__tests__/index.test.js

```diff
@@ -1,4 +1,4 @@
-const querystring = require('querystring');
+const validate = require('har-validator');
```

Comment:
FYI, most of the changes to this file were to rewrite some tests into using `it.each` to make them easier to maintain.